homeassistant/components/modbus/const.py

```diff
@@ -53,6 +53,15 @@
 CONF_SWAP_WORD = "word"
 CONF_SWAP_WORD_BYTE = "word_byte"
 CONF_TARGET_TEMP = "target_temp_register"
+CONF_HVAC_MODE_REGISTER = "hvac_mode_register"
```

Comment:
We should max. define HVAC_MODE, and use the consts already defined in components/climate for each entry.

Reply:
Done.